main.go

```diff
@@ -54,6 +54,7 @@ var (
 
 	samplesQueueCapacity = flag.Int("storage.incoming-samples-queue-capacity", 64*1024, "The capacity of the queue of samples to be stored. Note that each slot in the queue takes a whole slice of samples whose size depends on details of the scrape process.")
 
+	chunkType       = flag.Int("storage.local.chunk-type", 1, "Which chunk encoding version to use. Currently supported is 0 and 1.")
```

Comment:
Ok, reading further I guess you need this configurability at least in the code for now to be able to test and benchmark both cases. Humm... ok. But does it need a flag?